frontend/src/features/properties/selector/MapSelectorContainer.test.tsx

```diff
@@ -1,85 +0,0 @@
-import { PropertyPopUpContext } from 'components/maps/providers/PropertyPopUpProvider';
```

Comment:
This is a real shame, I like a lot of the changes here but losing out on test coverage hurts, and doesn't feel like a good use of either of our time. I think we should get together and discuss what happened here so we can avoid something like this from happening in the future.